src/server/system_services/bucket_server.js

```diff
@@ -539,6 +541,8 @@ async function read_bucket_sdk_info(req) {
             .then(get_bucket_info),
     };
 
+    reply.force_md5_etag = bucket.force_md5_etag;
```

Comment:
you should return `force_md5_etag` in `get_bucket_info` and not here. this implementation does not match the `bucket_api` definition (`force_md5_etag` is defined inder `bucket_info`)